### PR TITLE
from a record of one table

### PR DESCRIPTION
**Pull request by franzholz**

### CODE REVIEW

File: Documentation/ColumnsConfig/Type/Group/Index.rst [hunk 1]
```diff
@@ -6,23 +6,23 @@
 Group fields
 ============
 
-The group element in TYPO3 makes it possible to create references to records from multiple tables in the system.
+The group element (:php:`type' == 'group'`) in TYPO3 makes it possible to create references from a record of one table to many records from multiple tables in the system.
```
Comment (mbrodala) [inline Documentation/ColumnsConfig/Type/Group/Index.rst:9]:
```suggestion
The group element (:php:`type' => 'group'`) in TYPO3 makes it possible to create references from a record of one table to many records from multiple other tables in the system.
```

**Review by mbrodala — CHANGES_REQUESTED** [review]

File: Documentation/ColumnsConfig/Type/Group/Index.rst (hunk 1 above)
Comment (franzholz) [reply Documentation/ColumnsConfig/Type/Group/Index.rst:9]: It can also be a related between to tt_content tables. So both tables can be the same.

**Review by franzholz — COMMENTED** [review]
Help the programmer to find the enable method.

File: Documentation/ColumnsConfig/Type/Group/Index.rst (hunk 1 above)
Comment (mbrodala) [reply Documentation/ColumnsConfig/Type/Group/Index.rst:9]: True, I've adjusted the suggestion to make this clearer.

Comment (franzholz) [reply Documentation/ColumnsConfig/Type/Group/Index.rst:9]: instead of "The foreign table(s)"
better:
The foreign table

This would be understandable.

Comment (mbrodala) [reply Documentation/ColumnsConfig/Type/Group/Index.rst:9]: Well, there can be many foreign tables ...

Comment (franzholz) [reply Documentation/ColumnsConfig/Type/Group/Index.rst:9]: But please remove the (s). This does not look good.

Comment (mbrodala) [reply Documentation/ColumnsConfig/Type/Group/Index.rst:9]: Sure, done now. :+1:

**Review by mbrodala — APPROVED** [review]

**Pull request merged**